playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -64,6 +64,7 @@ class RelatedPod(BaseModel):
     addresses: str
     containers: List[RelatedContainer]
     status: Optional[str] = None
+    status_details: Optional[str] = None
```

Comment:
I think the field was added to the wrong class
This field is on `RelatedPod`, but should be on `RelatedContainer`, right?

Reply:
Obviously, that was a mistake.

Reply:
BTW `status_details` for `RelatedPods` have also been implemented, as discussed.